src/main/java/transforms/ArithmeticTransform.java

```diff
@@ -0,0 +1,109 @@
+     * @param alternative:     if left or right transformation fail to parse to double: what should the value be (defaults to null);
+     */
+    public ArithmeticTransform(
+            @JsonProperty( Constants.LEFTTRANSFORMS ) List<Transformation> leftTransforms,
+            @JsonProperty( Constants.RIGHTTRANSFORMS ) List<Transformation> rightTransforms,
+            @JsonProperty( Constants.OPERATOR ) String operator,
+            @JsonProperty( Constants.ELSE ) Optional<Double> alternative
+    ) {
+        this.operator = operator;
+        this.leftTransforms = leftTransforms;
+        this.rightTransforms = rightTransforms;
+        this.alternative = alternative.orElse( null );
+    }
+
+    @JsonProperty( Constants.OPERATOR )
+    public String getOperator() {
+        return operator;
+    }
+
+    @JsonProperty( Constants.LEFTTRANSFORMS )
+    public List<Transformation> getLeftTransforms() {
+        return leftTransforms;
+    }
+
+    @JsonProperty( Constants.RIGHTTRANSFORMS )
+    public List<Transformation> getRightTransforms() {
+        return rightTransforms;
+    }
+
+    public Double getValueFromTransform( Object transformed ) {
+
+        if ( StringUtils.isBlank( String.valueOf( transformed ) ) ) {
+            logger.debug( "Unable to parse number {} for arithmetic transform, returning",
+                    String.valueOf( transformed ),
+                    this.alternative );
+            return this.alternative;
+        } else {
+            try {
+                return Double.parseDouble( String.valueOf( transformed ) );
+            } catch ( Exception e ) {
+                logger.debug( "Unable to parse number {} for arithmetic transform, returning",
+                        String.valueOf( transformed ),
+                        this.alternative );
+                return this.alternative;
+            }
+        }
+    }
+
+    @Override
+    public Object apply( Map<String, String> row ) {
+
+        Object leftTransformed = row;
+        for ( Transformation t : leftTransforms ) {
+            leftTransformed = t.apply( leftTransformed );
+        }
+        Double left = getValueFromTransform( leftTransformed );
+
+        Object rightTransformed = row;
+        for ( Transformation t : rightTransforms ) {
+            rightTransformed = t.apply( rightTransformed );
+        }
+        Double right = getValueFromTransform( rightTransformed );
+
+        if ( left == null | right == null ) {
+            return null;
+        }
+
+        switch ( operator ) {
+            case "+":
+                return left + right;
+            case "-":
+                return left - right;
+            case "*":
+                return left * right;
+            case "/":
+                return left / right;
+            default:
+                throw new IllegalStateException( String
+                        .format( "Unknown operator in ArithmeticTransformation: ", operator ) );
```

Comment:
You need a `{}` at the end of your string to make use of that `operator` argument